packages/frontend/src/presenter/ApiDefs/useWebhooks.ts

```diff
@@ -0,0 +1,14 @@
+import { useFieldArray } from 'react-hook-form';
+
+export const useWebhooks = ({ control }: any) => {
```

Comment:
We have general ControlType `({ control }: any)` -> `({ control }: ControlType)`